src/lib/components/cloudprogressbar/CloudProgressBar.component.js

```diff
@@ -0,0 +1,160 @@
+//@flow
+import React from 'react';
+import styled, { css } from "styled-components";
+import * as defaultTheme from "../../style/theme";
+
+export type CloudProgressBarProps = {
+  percentage: number,
+  borderSize: string,
+  color?: string,
+  progressColor?: string,
+  label?: string,
+  fontCOlor?: string,
+  fontSizePercentage?: string,
+  fontSizeLabel?: string
+};
+
+const Container = styled.div`
+  ${props => {
+    return css`
+      width: 100%;
+      text-align: center;
+      position: relative;
+    `;
+  }}
+`;
+
+const ContainerProgress = styled.div`
+  ${props => {
+    return css`
+      @keyframes heightAnimation {
+        from {
+          height: 100%;
+        }
+        to {
+          height: ${100 - props.percentage}%;
+        }
+      }
+      animation-duration: 1s;
+      animation-fill-mode: both;
+      animation-name: heightAnimation;
+      position: absolute;
+      z-index: 222;
```

Comment:
Because we have "two layers" on the component, one is of complete cloud and other is of the progress area, so without this `z-index` the component doesn't work how expected.

Reply:
I understand your point. But feel free to remove z-index to give a try.

IMO, in `<Cloud>` we use `progressCloudColor` which is the color of the complete cloud at the beginning. But it will override by `<CloudProgress>` with the `cloudColor` which is the color of the bottom part of the cloud.

So I guess using `z-index` is also doable.

Reply:
Nice! You was right, we don't need of `z-index`.